src/main/java/com/iotiq/application/config/DbMigrator.java

```diff
@@ -0,0 +1,33 @@
+package com.iotiq.application.config;
+
+import org.flywaydb.core.Flyway;
+
+import javax.sql.DataSource;
+import java.util.Collection;
+
+public class DbMigrator {
+    private static DbMigrator instance;
+
+    private DbMigrator() {
+    }
+
+    public static DbMigrator getInstance() {
+        if (instance == null) {
+            instance = new DbMigrator();
+        }
+        return instance;
+    }
```

Comment:
Why are we implementing the singleton pattern here? Can you not use this as a component or service?

Reply:
I refactored it as component